react-client-app/src/App.jsx

```diff
@@ -14,6 +14,7 @@ import ScanReportAdminForm from './views/ScanReportAdminForm'
 import Home from './components/Home';
 import { getScanReportConcepts, m_allowed_tables, useDelete, useGet, usePost, mapConceptToOmopField, saveMappingRules } from './api/values'
 import UploadScanReport from './components/UploadScanReport'
+import DatasetTbl from './components/DatasetTbl'
```

Comment:
Same as with #437 and #434, since this "component" is essentially a whole page and not a  reusable piece used on multiple different pages, could this go in `./views/` instead of `./components/` pls?